### PR TITLE
Integrating RRT Connect path planner from OMPL

### PR DESCRIPTION
**Pull request by kvmanohar22**
Currently a simple `isValidState` is implemented. This will be replaced with `LSTM` predictions from the learned model in near future.

### CODE REVIEW

**Comment by annimesh2809** [comment]
Seems like the build failed. We should fix that before merging.

**Comment by kvmanohar22** [comment]
The problem seems to occur because `cmake` is unable to find the `ssl_robotConfig.cmake` which is present in `$ENV{HOME}/catkin_ws/devel`. Apparently this variable is not set in travis (see [here](https://travis-ci.org/krssg-ssl/navigation/builds/239446758#L8574)) but is added to `CMAKE_PREFIX_PATH` in my system :laughing:. One way to get around this is to set this path manually !

**Comment by MayankB11** [comment]
Shouldn't each bot be able to use the planner at the same time ? In this case, while one bot requests the service other bots will get queued.

File: src/planners/ompl_planner.cpp [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+Planner::~Planner() {
+
+}
+
+bool Planner::isStateValid(const ob::State *state) {
+  const ob::SE2StateSpace::StateType *state_2d= state->as<ob::SE2StateSpace::StateType>();
+  const double &x(state_2d->getX()), &y(state_2d->getY());
+
+  for (int i = 0; i < numObstacles; ++i){
+    if (sqrt(pow((obstacles[i].x - x), 2)+pow((obstacles[i].y - y), 2)) <= 10.0){
```
Comment (annimesh2809) [inline src/planners/ompl_planner.cpp]: Why is 10.0 hard-coded here?

Comment (MayankB11) [reply src/planners/ompl_planner.cpp]: that should be changed according to grSim or camera input

Comment (kvmanohar22) [reply src/planners/ompl_planner.cpp]: In fact `isStateValid()` function will be rewritten once we add the functionality of `LSTM`.

**Comment by kvmanohar22** [comment]
This failure seems to arise because ompl is not installed, we need to add these installation commands in `install.sh`

**Pull request merged**